lib/src/components/slider.dart

```diff
@@ -142,51 +144,60 @@ class _ShadSliderState extends State<ShadSlider> {
     final effectiveThumbRadius =
         widget.thumbRadius ?? theme.sliderTheme.thumbRadius ?? 10.0;
 
-    return Theme(
-      data: mTheme.copyWith(
-        sliderTheme: mTheme.sliderTheme.copyWith(
-          trackHeight: effectiveTrackHeight,
-          thumbShape: ShadSliderThumbShape(
-            radius: effectiveThumbRadius,
-            borderColor: effectiveThumbBorderColor,
-            disabledBorderColor: effectiveDisabledThumbBorderColor,
-            thumbColor: effectiveThumbColor,
+    return Focus(
+      onFocusChange: (focused) {
+        setState(() {
+          isFocused = focused;
+        });
+      },
+      skipTraversal: true,
```

Comment:
Don't use the Focus widget, use the ShadFocusable instead. No need to use setState.
Pass the widget.focusNode to it, and in the builder you get the `focused` value.
Merge the new main to have all the properties like `skipTraversal`